example_config.yml

```diff
@@ -8,4 +8,3 @@ objective:
 
 max-runs: 50
 n-repetitions: 1
-n-jobs: 15
```

Comment:
I removed this as it seems to limit `skglm` to one iteration

Reply:
I don't think this is related, can you isolate this